### ISSUE TITLE
Add Summary & Breakdown of Test Result Performance to AppVeyor Output

### ISSUE THREAD

**Issue by PlagueHO** [PowerShell/DscResource.Tests#309]
As a DSC resource kit contributor
I want to be able to see the amount of time spent running tests in each file/fixture
So that my test I can identify tests that might be targets for improving performance

See: https://github.com/PowerShell/xPSDesiredStateConfiguration/issues/477#issuecomment-457860098

Tagging @mhendric, @johlju

**Comment by johlju** [comment]
I liked the idea of outputing a table from the NUnitXML. Start with that and see how far it gets us? I think it should be enough. 🤔

**Comment by johlju** [comment]
Maybe instead of outputting a table, maybe we can add the information to the test pane? If we made this into a helper function, when looping through the NUnitXML we could just add rows to the test pane. That way each file would be at the top of the test pane results. 🤔 
I'm good with an ASCII table in the output view too 🙂 

https://github.com/PowerShell/DscResource.Tests/blob/bfa3d0d8e0a58919547d4028801ae3222c34dc4c/AppVeyor.psm1#L865-L892

**Comment by PlagueHO** [comment]
Do you mean the AppVeyor Tests:
![image](https://user-images.githubusercontent.com/7589164/51806168-69d10900-22db-11e9-920a-90ac7e5fd98a.png)

So you're thinking adding test summaries directly to the test results tab in AppVeyor for the run? That could definitely be worth investigating!

**Comment by johlju** [comment]
Yes exactly. If test name need to have a value then just use the filename there too. I'm thinking one entry per file, and after that all the entries is added from the pester object like it does today. Hopefully they are viewed in the same order they are added, otherwise it won't work :)
